ESP8266Interface.h

```diff
@@ -23,6 +23,10 @@
 
 #define ESP8266_SOCKET_COUNT 5
 
+#define SSID_MAX_LENGTH 32 /* 32 is what 802.11 defines as longest possible name; +1 for the \0 */
```

Comment:
As these are in a public header file, probably best to stick `ESP8266_` in front.

Or, as it's C++, make them possibly-private `const` members.

I'd personally be happy without any defines if all the code used `sizeof ap_pass` etc, then the magic number would be in exactly one place, which I think is sufficient.

Reply:
`sizeof`would be nice if there was not a minimum length for the password.

Reply:
True, but then the minimum also doesn't need to be in your public header. Still, I can see merit in having min and max declared near each other.